modules/apps/imageio-plugins/imageio-plugins/build.gradle

```diff
@@ -3,21 +3,21 @@ dependencies {
 	compileInclude group: "com.twelvemonkeys.common", name: "common-image", version: "3.3.2"
 	compileInclude group: "com.twelvemonkeys.common", name: "common-io", version: "3.3.2"
 	compileInclude group: "com.twelvemonkeys.common", name: "common-lang", version: "3.3.2"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-bmp", version: "3.4.3"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-core", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-hdr", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-icns", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-iff", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-jpeg", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-metadata", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-pcx", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-pict", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-pnm", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-psd", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-sgi", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-tga", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-thumbsdb", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-tiff", version: "3.5"
+	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-bmp", version: "3.8.2"
```

Comment:
@jeffwu0724 These three seem related to others, can you also upgrade them?

	compileInclude group: "com.twelvemonkeys.common", name: "common-image", version: "3.3.2"
	compileInclude group: "com.twelvemonkeys.common", name: "common-io", version: "3.3.2"
	compileInclude group: "com.twelvemonkeys.common", name: "common-lang", version: "3.3.2"